charts/hami/templates/device-plugin/daemonsetvalidator-configmap.yaml

```diff
@@ -0,0 +1,48 @@
+{{- if .Values.devicePlugin.autoValidatorDriver }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: driver-validator-configmap
+data:
+  validator.sh: |
+    #!/bin/zsh
+    if [ -f "/host/usr/bin/nvidia-smi" ]; then
+        driverRoot="/host"
+        isHostDriver="true"
+    else
+        driverRoot="/run/nvidia/driver"
+        isHostDriver="false"
+    fi
+
+    # check this directory /run/nvidia/driver does exist?
+    if [ "$isHostDriver" = "false" ]; then
+      if [ ! -d "/run/nvidia/driver" ]; then
+        echo current node $NODE_NAME not GPU device; sleep infinity
```

Comment:
Why do we need to `sleep infinity` here?

Reply:
because current NODE not GPU device, so sleep infinity.

Reply:
If the driver-validator is installed before the `gpu-operator`, wouldn't this shell script be unusable all along?